swagger.yaml

```diff
@@ -1,10 +1,10 @@
-openapi: '3.0.3'
+openapi: "3.0.3"
 info:
-  title: 'OPUS'
-  version: '0.1'
+  title: "OPUS"
+  version: "1.0"
```

Comment:
might switch back to v0.1 @QAston , we're still missing full implementation from `/validators` endpoint & there's going to be a signed exit transaction endpoint, what do you think?

Reply:
yeah i decided to not update the version yet, because there's also a change coming up in the `/stake` request body, will promote to first version after: 

1. `/stake` request body was updated 
2. `/validators` endpoint was finalised
3. `/unstake` endpoint was integrated with portal (potentially leading to changes as well)